rozdzial2.tex

```diff
@@ -48,7 +48,7 @@ \section{Kompilacja}\label{sec:kompilacja}
 ps2pdf test.ps
 \end{lstlisting}
 %
-lub za pomocą PDF\LaTeX{}:
+lub za pomocą PDF\LateX:
```

Comment:
Tutaj pojawia się jeszcze jedno ostrzeżenie:

> Intersentence spacing (`\@') should perhaps be used. (13) 

Wydaje mi się w tym wypadku wstawienie `\@` nieszkodliwe - ponieważ po : i tak jest nowa linia.

Reply:
Jest nieszkodliwe, ale właśnie z powodu że po : jest nowa linia, nic to nie zmieni. Ponadto stosowanie w takim przypadku `\@` jak i `\ ` po skrócie `itp.\ ` jest niepotrzebne, gdyż w polskiej typografii, podobnie jak w francuskiej stosuje się równej długości odstępy międzyzdaniowe i międzysłowne (zapewnia to makro `\frenchspacing` w aghdpl.cls). Zapomniałem o tym, a to oznacza że te ostrzeżenia też są bez znaczenia, a nie, tak jak napisałem wcześniej, istotne.

Zmieniłeś wszystkie `\LaTeX` na `\LateX`. Teraz się nawet nie kompiluje...